ci/gpu_test.sh

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env bash
+# Copyright (c) 2018-2022, NVIDIA CORPORATION.
+##########################################
+# cuGraph GPU build & testscript for CI  #
+##########################################
+set -e           # abort the script on error, this will change for running tests (see below)
+set -o pipefail  # piped commands propagate their error
+NUMARGS=$#
+ARGS=$*
+
+function hasArg {
+    (( ${NUMARGS} != 0 )) && (echo " ${ARGS} " | grep -q " $1 ")
+}
+
+# Set path, build parallel level, and CUDA version
+export PATH=/opt/conda/bin:/usr/local/cuda/bin:$PATH
+export PARALLEL_LEVEL=${PARALLEL_LEVEL:-4}
+export CUDA_REL=${CUDA%.*}
+export CONDA_ARTIFACT_PATH=${WORKSPACE}/ci/artifacts/cugraph/cpu/.conda-bld/
```

Comment:
`CONDA_ARTIFACT_PATH` is a remnant of Project Flash and therefore should be removed.